### PR TITLE
Always allow enabling sending read receipts

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-web/issues/23433

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Always allow enabling sending read receipts ([\#9367](https://github.com/matrix-org/matrix-react-sdk/pull/9367)). Fixes vector-im/element-web#23433.

### CODE REVIEW

File: test/components/views/settings/tabs/user/PreferencesUserSettingsTab-test.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+
+/*
+Copyright 2022 Šimon Brandner <simon.bra.ag@gmail.com>
```
Comment (turt2live) [inline test/components/views/settings/tabs/user/PreferencesUserSettingsTab-test.tsx:3]:
```suggestion
/*
Copyright 2022 Šimon Brandner <simon.bra.ag@gmail.com>
```

also not sure this is the right copyright

**Comment by turt2live** [comment]
single comment wasn't supposed to clear review - thanks github :|

File: test/components/views/settings/tabs/user/PreferencesUserSettingsTab-test.tsx (hunk 1 above)
Comment (turt2live) [reply test/components/views/settings/tabs/user/PreferencesUserSettingsTab-test.tsx]: @SimonBrandner fix still has an extra blank line on it

File: test/components/views/settings/tabs/user/PreferencesUserSettingsTab-test.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
+
```
Comment (SimonBrandner) [inline test/components/views/settings/tabs/user/PreferencesUserSettingsTab-test.tsx:1]:
```suggestion
```

**Review by t3chguy — APPROVED** [review]

**Pull request merged**